Resources/Prototypes/Entities/Effects/Markers/spawn_points.yml

```diff
@@ -27,3 +26,13 @@
   - type: SpawnPoint
     spawn_type: Observer
 
+- type: entity
+  name: centcom official spawn point
+  id: SpawnPointCentcomOfficial
+  parent: SpawnPoint
+  components:
+  - type: GhostRoleMobSpawnerComponent
+  - type: Sprite
+    state: Centcom
+  - type: SpawnPoint
+    job_id: SpawnPoint
```

Comment:
This is not how ghost roles work.
Ghost roles don't need nor use the SpawnPoint component.
This specific ghost role component takes the following variables:
- prototype: The prototype that will be used for the mob.
- deleteOnSpawn: whether this should be deleted after someone takes over the role.
- availableTakeovers: How many times this ghost role can be taken over. Defaults to 1, a new entity will be created per takeover.
- makeSentient: Whether the mob should be made sentient on spawn.
- name: The name for the ghost role on the ghost role window
- description: The description for the ghost role on the ghost role window